### PR TITLE
test-configs.yaml: update Debian rootfs URLs

### PR DESCRIPTION
**Pull request by gctucker**
Update all the Debian rootfs URLs to 20210520.0.

Signed-off-by: Guillaume Tucker <guillaume.tucker@collabora.com>

### CODE REVIEW

**Review by alexandrasp — APPROVED** [review]
LGTM

**Comment by gctucker** [comment]
There appears to be an issue, at least with the kselftest image:
https://lava.collabora.co.uk/scheduler/job/3861706

When resubmitted with the regular Debian it works fine:
https://lava.collabora.co.uk/scheduler/job/3864208

File: config/core/test-configs.yaml [hunk 1]
```diff
@@ -33,36 +33,36 @@ file_systems:
...
   debian_buster-igt_ramdisk:
     type: debian
-    ramdisk: 'buster-igt/20210503.0/{arch}/rootfs.cpio.gz'
+    ramdisk: 'buster-igt/20210520.0/{arch}/rootfs.cpio.gz'
 
   debian_buster_kselftest:
     type: debian
-    ramdisk: 'buster-kselftest/20210514.0/{arch}/rootfs.cpio.gz'
-    nfs: 'buster-kselftest/20210514.0/{arch}/full.rootfs.tar.xz'
+    ramdisk: 'buster-kselftest/20210520.0/{arch}/rootfs.cpio.gz'
```
Comment (alexandrasp) [inline config/core/test-configs.yaml:54]: @gctucker the problem is here, ramdisk can not be rootfs, it needs to initrd, right? ;)

Comment (alexandrasp) [reply config/core/test-configs.yaml:54]: https://lava.collabora.co.uk/scheduler/job/3871673

Comment (gctucker) [reply config/core/test-configs.yaml:54]: Yes, good point.  Please send a fix for that and I'll skip the revert PR #692.

**Pull request merged**